### PR TITLE
Formatting and license

### PR DESCRIPTION
**Pull request by keveleigh**
Added some missing licenses from MSFT contributors in #120 and #122 as well as taking this opportunity to run a quick format.

### CODE REVIEW

**Comment by satyapoojasama** [comment]
Thanks for this PR Kurtis!

**Review by satyapoojasama — APPROVED** [review]

File: BasicSample/Assets/Rendering/Scripts/CubesGrid.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
-using System.Collections;
```
Comment (satyapoojasama) [inline BasicSample/Assets/Rendering/Scripts/CubesGrid.cs:1]: Is removing these headers intentional?

File: BasicSample/Assets/Shared/Scripts/HideHandSpheresWhenNotTracked.cs [hunk 2]
```diff
@@ -1,8 +1,9 @@
-using System.Collections.Generic;
```
Comment (satyapoojasama) [inline BasicSample/Assets/Shared/Scripts/HideHandSpheresWhenNotTracked.cs:1]: same here

File: BasicSample/Assets/Rendering/Scripts/CubesGrid.cs (hunk 1 above)
Comment (keveleigh) [reply BasicSample/Assets/Rendering/Scripts/CubesGrid.cs:1]: Yep! The auto-formatter removed any unused `using` statements.

Comment (satyapoojasama) [reply BasicSample/Assets/Rendering/Scripts/CubesGrid.cs:1]: cool, what auto-formatter are you using?

Comment (keveleigh) [reply BasicSample/Assets/Rendering/Scripts/CubesGrid.cs:1]: Mostly the in-box Visual Studio one (I'm pretty sure, at least)! I tend to use the right-click menu in the Solution Explorer:

image


You can also do a single file with 
image, though I believe this option cleans up slightly fewer things by default.

Comment (satyapoojasama) [reply BasicSample/Assets/Rendering/Scripts/CubesGrid.cs:1]: This is good to know, always been using VSCode and did not realize Visual Studio has this feature. Also, its been difficult to auto format C# code, this is great! Will add this to best practices documentation.

**Pull request merged**